source/game/item/ItemDirector.cc

```diff
@@ -4,34 +4,40 @@
 
 namespace Item {
 
+/// @addr{0x80799794}
 void ItemDirector::init() {
     for (auto &kart : m_karts) {
         kart.inventory().setItem(ItemId::TRIPLE_MUSHROOM);
     }
 }
 
+/// @addr{0x80799850}
 void ItemDirector::calc() {
     for (auto &kart : m_karts) {
         kart.calc();
     }
 }
 
+/// @addr{0x80799138}
 ItemDirector *ItemDirector::CreateInstance() {
     assert(!s_instance);
     s_instance = new ItemDirector;
     return s_instance;
 }
 
+/// @addr{0x80799188}
 void ItemDirector::DestroyInstance() {
     assert(s_instance);
     delete s_instance;
     s_instance = nullptr;
 }
 
+/// @addr{0x809C3618}
```

Comment:
Right, in these kinds of instances I figured it would be helpful to associate the doxygen address of our non-emitted getter with the singleton's address. Fine if you don't want to set that precedent

Reply:
I would prefer otherwise because there *are* instances where getters have addresses (80008e84, for example).